android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -114,11 +115,78 @@ constructor(
       ?.also { configsLoadedCallback(true) }
   }
 
+  suspend fun loadConfigurationsLocally(appId: String, configsLoadedCallback: (Boolean) -> Unit) {
+    val parsedAppId = appId.substringBefore("/$DEBUG_SUFFIX")
+    this.appId = parsedAppId
+
+    // appId is identifier of Composition
+    val baseConfigPath = BASE_CONFIG_PATH.run { replace(DEFAULT_APP_ID, parsedAppId) }
+
+    runCatching {
+      context
+        .assets
+        .open(baseConfigPath.plus(COMPOSITION_CONFIG_PATH))
+        .bufferedReader()
+        .use { it.readText() }
+        .decodeResourceFromString<Composition>()
+        .section
+        .forEach { sectionComponent ->
+          // each section in composition represents workflow
```

Comment:
what's up w/this, remove?

Reply:
i thought it was necessary.
yeah, let's avoid repetitiveness

Reply:
cool yea I think you can delete the commented code if we don't need.

I was wondering if it's commented and waiting to be uncommented in this PR

Reply:
got it